Add deprecation note (#1485)
hyperledger/besu-docs: docs/public-networks/how-to/troubleshoot/trace-transactions.md

# Trace transactions

:::caution Forest pruning deprecation notice

> Forest pruning (the `pruning-enabled` option) is being deprecated and will be removed in a future release. This option is no longer being maintained. Using Bonsai Tries as an alternative for saving disk space is recommended.

Reviewer: ```suggestion
Forest pruning (using the [`--pruning-enabled`](../../../reference/cli/options.md#pruning-enabled) option) is deprecated and will be removed in a future release. We recommend using [Bonsai Tries](../../../concepts/data-storage-formats.md#bonsai-tries) as an alternative for saving disk space.
```

https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/r/recommend

Same for the other admonitions.
